buildsystem/HandlePythonOptions.cmake

```diff
@@ -7,17 +7,10 @@ find_package(Python 3.4 REQUIRED)
 find_package(Cython 0.25 REQUIRED)
 find_package(Numpy REQUIRED)
 
-py_get_config_var(OPT PYEXT_CXXFLAGS)
```

Comment:
depending on the platform, this would yield important flags that we otherwise could not anticipate. so i wouldn't remove it.